src/components/common/Input/styles.js

```diff
@@ -1,8 +1,41 @@
 import { StyleSheet } from 'react-native';
 
+const baseLabel = {
+  height: 15,
+  textAlign: 'center',
+  fontWeight: 'bold'
+};
+
+const baseInput = {
+  width: 230,
+  height: 40,
+  borderStyle: 'solid',
+  borderWidth: 1,
+  padding: 5,
+  fontWeight: 'bold',
```

Comment:
Just a small detail, but I'd make sure we consistently either have or don't have a `,` on every last element. Not sure if there's a standard for this, tho.

Reply:
I think that depends on the linter.